tools/generate_hotkeys.py

```diff
@@ -38,6 +38,10 @@ def lint_hotkeys_file() -> None:
             various_key_combinations = various_key_combination(key_combinations_list)
             # Check description style
             if not re.match(help_text_style, help_text):
+                various_key_combinations = various_key_combinations.replace("<kbd>", "")
+                various_key_combinations = various_key_combinations.replace(
+                    "</kbd>", ""
+                )
```

Comment:
Here you can do both replaces one after the other, like so.
`str.replace().replace()`